app/models/taxon_parent_links_update.rb

```diff
@@ -0,0 +1,11 @@
+class TaxonParentLinksUpdate
```

Comment:
Is two classes perhaps overkill, since we're only going to clear the `parent` once?